scripts/qa_crowbarsetup.sh

```diff
@@ -31,7 +31,7 @@ wanthyperv=
 
 export cloudfqdn=${cloudfqdn:-$cloud.cloud.suse.de}
 export nodenumber=${nodenumber:-2}
-export tempestoptions=${tempestoptions:--t -s}
+export tempestoptions=${tempestoptions:---subunit smoke}
```

Comment:
it looks like you always want to use --subunit then, so it doesn't make sense imho to add that to the tempestoptions. 

tempestoptions should then be "smoke" or  "full" I guess.